src/importer.cpp

```diff
@@ -349,27 +476,27 @@ IssuePtr Importer::ImporterImpl::makeIssueCyclicDependency(const ModelPtr &model
     }
     auto issue = Issue::create();
     issue->setDescription(msg);
-    issue->setLevel(Issue::Level::WARNING);
+    issue->setLevel(Issue::Level::ERROR);
     issue->setReferenceRule(Issue::ReferenceRule::IMPORT_EQUIVALENT);
-    std::vector<std::tuple<std::string, std::string, std::string>>().swap(history);
+    HistorySearchVector().swap(history);
```

Comment:
I feel like it's a way to empty `history`, right? If so, why do we need to do that? Also, if we need to empty `history`, then why not simply call `history.clear()`?

Reply:
a107cfd4